tex-nutshell/tex-nutshell.tex

```diff
@@ -330,7 +330,7 @@
 define it by
 \i def `\def<character>{...}`.
 If a `<control sequence>` is
-listed in this document, the active character is meant too.
+listed in this document, then it is also defined as active character.
```

Comment:
I find both versions still not fully clear. My try:

`From now on, "control sequence" means either a normal control sequence or an active character.`

What do you both think? @olsak @itmm